### PR TITLE
fix: #24519 make upload action buttons focusable

### LINKED ISSUE #24519: Upload control remove file button is not keyboard accessible
upload_removebutton
- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[Edit on CodeSandbox](https://codesandbox.io/s/sfbuw?file=/index.js)

### Steps to reproduce
1. Upload any file in the upload control 
2. Use keyboard "tab" key to navigate to remove file button for the uploaded file
3. It seems that unable to remove file using keyboard "tab" key and the focus not set on uploaded file

### What is expected?
Upload control should be keyboard accessible 

### What is actually happening?
Unable to focus the remove file button on uploaded file

| Environment | Info |
|---|---|
| antd | 4.2.5 |
| React | 16.12.0 |
| System | Windows 10 |
| Browser | Edge Version 81.0.416.72 (Official build) (64-bit) or Chorme Version 81.0.4044.138 (Official Build) (64-bit) |

### PR DESCRIPTION
**Pull request by morenyang**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

close #24519 

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Perfermance optimization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

https://github.com/ant-design/ant-design/issues/24519

### 💡 Background and solution

- https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Keyboard
- change `tabindex` on icon to `0`
- update `opacity` control to action-icon

ezgif-3-7a03c8904ad8

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | Fix Upload delete icon cannot be navigate via keyboard. |
| 🇨🇳 Chinese | 解决 Upload 删除图标键盘无法导航操作的问题。 |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by morenyang** [comment]
CI failed with message: 
```
The error below may be caused by using the wrong test environment, see https://jestjs.io/docs/en/configuration#testenvironment-string.
Consider using the "jsdom" test environment.
    
ReferenceError: window is not defined
```

File: components/upload/UploadList.tsx [hunk 1]
```diff
@@ -201,7 +201,11 @@ export default class UploadList extends React.Component<UploadListProps, any> {
               () => this.handleClose(file),
               locale.removeFile,
             )) || (
-            <DeleteOutlined title={locale.removeFile} onClick={() => this.handleClose(file)} />
+            <DeleteOutlined
+              title={locale.removeFile}
+              onClick={() => this.handleClose(file)}
+              tabIndex={0}
+            />
```
Comment (afc163) [inline components/upload/UploadList.tsx:208]: 包一个 a 或者 button 标签应该就行了，加 tabIndex 是下策。

Comment (afc163) [reply components/upload/UploadList.tsx:208]: image

好像也可行。

**Comment by afc163** [comment]
changelog 要从用户角度写，对于这个问题应该是：

- 解决 Upload 删除图标键盘无法导航操作的问题。
- Fix Upload delete icon cannot be navigate via keyboard.

File: components/upload/UploadList.tsx (hunk 1 above)
Comment (morenyang) [reply components/upload/UploadList.tsx:208]: 我一开始也是这么想的，但是怕改结构你们会有什么 concern ，所以就选 tabindex 这种改动比较小的方案

Comment (morenyang) [reply components/upload/UploadList.tsx:208]: Seems you are right, 我选上以后没法删除...

**Pull request merged**